app/Main.hs

```diff
@@ -359,8 +372,8 @@ requestMountInfo context =
     request
         = setRequestManager
               (cHttpManager context)
-        $ setRequestHeader "x-vault-token"
-              [SBS.pack (getOptionsValue oVaultToken cliOptions)]
+        $ addVaultToken cliOptions
+        $ setRequestHeader "x-vault-request" [SBS.pack "true"]
```

Comment:
The `SBS.pack` is not needed here, because [`BytesString` implements `IsString`](https://www.stackage.org/haddock/lts-16.29/bytestring-0.10.10.1/Data-ByteString.html#t:ByteString), and because `OverloadedStrings` is enabled at the top of this file. With `OverloadedStrings` enabled, string literals can have any type `IsString a => a`, not just `String`, so `"true"` can be a `ByteString` literal.